src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/OperatorWrapCheck.java

```diff
@@ -308,4 +316,9 @@ else if (option == WrapOption.EOL
         }
     }
 
+    @Override
+    public boolean isCommentNodesRequired() {
+        return true;
```

Comment:
Why is this needed? If this is needed, check may become more complicated to validate as now we need to determine if comments showing up in the AST tree is causing a separate issue.

Reply:
If we donot set this true then there will be no comment nodes found so how will i check where the tree contains a comment node?

Reply:
Do you need to know more about comments more than reported issue, which is a line ending with a comment? Since we are reading the source why can't we check for `//` or `/*`? Will we have further issues with `/*` since more code can be after `*/`?

If we need to add this method, we will need to split the work into 2 commits and provide regression between each one.

1st commit should be adding this method and any support now that this method is added and regression should expect no differences.
2nd commit is to fix the original issue and regression should show differences.

Reply:
Okay so I will just check for `//` and `/*` in the lines @rnveach this will be much better

Reply:
@harsh-kukreja If any good violations are removed from regression because of incorrect handling of  `/*`, we might have to fix it.